### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by cwaku**
- Add styling to contact form
- Submit the form to Formspree
- Refactor codebase
- Make changes to overall responsiveness for mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -174,12 +175,32 @@ <h2>About</h2>
...
+    </p>
+  </div>
+    <span class="d-flex flex-cols input-wrapper margin-tp">
+      <label for="name" class="poppins">Your name</label>
+      <input class="primary-color no-outline" type="text" name="name" id="name" placeholder="" title="Your name" maxlength="30" required>
+    </span>
+    <span class="d-flex flex-cols input-wrapper">
+      <label for="email" class="poppins">Your email</label>
+      <input class="primary-color no-outline" type="text" name="email" id="email" placeholder="" title="Your email" required>
+    </span>
```
Comment (ZeenatLawal) [inline index.html:197]: - [ ] It will be good if the input tag for the email has a type of email not text, so there won't be invalid submissions of email.

Comment (ZeenatLawal) [reply index.html:197]: - [ ] The placeholder attribute for both input tags are not being used, I suggest that it should be removed.

File: index.html [hunk 2]
```diff
@@ -174,12 +175,32 @@ <h2>About</h2>
...
+    <textarea class="input-wrapper no-outline primary-color" name="message" id="message" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+    <span class="form-submit poppins dark-bg">
+      <button type="submit" class="dark-bg">Start collaboration</button>
+    </span>
```
Comment (ZeenatLawal) [inline index.html:201]: - [ ] The Start collaboration button should follow the button styling: Active, Hover and Pressed (see design)

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @cwaku and @blessedjasonmwanza, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @cwaku  Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:


brent-rambo-thumbs-up


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**